test/fw/bin/pbs_compare_results

```diff
@@ -87,18 +90,27 @@ def generate_html_report():
     </body>
     </html>
     '''
-    with open('performance_test_report.html', 'w+') as fp:
+    htmlfile = filepath + '.html'
```

Comment:
I think you should check `if not filepath.endswith('.html')` then add `.html`, user might have given that.

Same for .csv also